### DISCUSSION TITLE
ImmortalWrt Firmware Selector: appears when compiling SNAPSHOT branching (Error: Missing Checksum) prompts

### DISCUSSION THREAD

**Discussion by dpy013** [immortalwrt/immortalwrt#1271] [Q&A]
以下提供使用的软件包列表：
zsh curl wget unzip alsa-utils autocore automount base-files block-mount busybox ca-bundle default-settings-chn dnsmasq-full dropbear fdisk firewall4 fstools grub2-bios-setup ipv6helper kmod-8139cp kmod-8139too kmod-ac97 kmod-amd-xgbe kmod-bnx2 kmod-button-hotplug kmod-e1000 kmod-e1000e kmod-forcedeth kmod-fs-f2fs kmod-fs-vfat kmod-i40e kmod-igb kmod-igbvf kmod-igc kmod-ixgbe kmod-ixgbevf kmod-nf-nathelper kmod-nf-nathelper-extra kmod-nft-offload kmod-pcnet32 kmod-r8101 kmod-r8125 kmod-r8168 kmod-sound-hda-codec-hdmi kmod-sound-hda-codec-realtek kmod-sound-hda-codec-via kmod-sound-hda-core kmod-sound-hda-intel kmod-sound-i8x0 kmod-sound-via82xx kmod-tg3 kmod-tulip kmod-usb-audio kmod-usb-hid kmod-usb-net kmod-usb-net-asix kmod-usb-net-asix-ax88179 kmod-usb-net-rtl8150 kmod-usb-net-rtl8152-vendor kmod-vmxnet3 kmod-wireguard libc libgcc libustream-openssl logd luci luci-app-opkg luci-i18n-opkg-zh-cn luci-compat luci-lib-base luci-i18n-base-zh-cn luci-lib-fs luci-lib-ipkg luci-lib-json luci-lib-jsonc luci-proto-ipv6 luci-proto-wireguard luci-theme-material luci-app-firewall luci-i18n-firewall-zh-cn luci-app-ddns-go luci-i18n-ddns-go-zh-cn luci-app-wireguard luci-app-vlmcsd luci-i18n-vlmcsd-zh-cn luci-app-ramfree luci-i18n-ramfree-zh-cn luci-app-diskman luci-i18n-diskman-zh-cn luci-app-alist luci-i18n-alist-zh-cn luci-app-adbyby-plus luci-i18n-adbyby-plus-zh-cn luci-app-passwall luci-i18n-passwall-zh-cn mkf2fs mtd netifd nftables opkg partx-utils ppp ppp-mod-pppoe pppoe-discovery procd procd-seccomp procd-ujail uci uclient-fetch urandom-seed urngd tailscale tcpdump wireguard-tools
问题重现步骤
1.1 首先[打开](https://firmware-selector.staging.immortalwrt.org)
1.2 然后编译分支选择SNAPSHOT 
1.3 最后执行编译步骤即可重现问题
ps:
错误是提示 Error: Missing Checksum

**Comment by 1715173329** [comment]
> failed to allocate 1482 blocks, out of space

So you added too many packages and resulted out of space.
You have to run the IB locally with rootfs resized.
